flexmeasures/ui/crud/assets.py

```diff
@@ -321,8 +325,31 @@ def status(self, id: str):
         asset = process_internal_api_response(asset_dict, int(id), make_obj=True)
         status_data = build_sensor_status_data(asset)
 
+        # add data abount max_jobs_num forecasting and scheduling jobs
+        redis_connection_err = None
+        scheduling_job_data, forecasting_job_data = list(), list()
+        try:
+            jobs_data = build_asset_jobs_data(asset)
+        except NoRedisConfigured as e:
+            redis_connection_err = e.args[0]
+        else:
+            max_jobs_num = 10
+            scheduling_job_data = [
+                job_data
```

Comment:
In one loop, you say `job` and in the other  `job_data`. But here we could say `jd` I believe, so the loops might become one-liners.